apps/portal/packages/app/src/styles.css

```
@@ -0,0 +1,584 @@
+  font-family: var(--font-mono);
+  font-weight: 700;
+  transition: all 0.2s ease;
+}
+
+button.MuiButton-containedPrimary:hover {
+  background-color: var(--helios-solar-hover);
+  box-shadow: 0 0 20px var(--theme-primary-glow);
+}
+
+div[class*='BackstageWarningPanel-panel'] {
+  /*background-color: var(--rose);*/
```

Stylelint (17.5.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)